src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Display Apple A1419 iMac 27 inches Retina 5K Monoblock(MNED2UA/A)"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27&quot;  Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__text card__text--code">
+        Product code: 195434
+      </p>
+
+      <div class="card__container card__container--rate">
```

Comment:
```suggestion
      <div class="card__container card__container-rate">
```
the same

Reply:
check for all elements

Reply:
I have fixed, but npm tests have been failed.    
![image](https://user-images.githubusercontent.com/119630749/209731465-28d0b3c9-8ca5-4e48-b7cf-b35e805e5ec5.png)